conda/meta.yaml

```diff
@@ -8,6 +8,7 @@ source:
 
 requirements:
   build:
+    - boost-cpp
```

Comment:
I'm a little uncomfortable that we are not handling the build dependencies consistently now.

Reply:
This is not new. TBB is installed via conda already -- albeit an "optional" dependency.

Reply:
Yes, I did think that we have the same thing with TBB. Maybe not so bad for this reason.